tests/embedding_net_test.py

```diff
@@ -96,9 +96,9 @@ def test_iid_embedding_api(num_trials, num_dim):
 
 
 @pytest.mark.slow
-@pytest.mark.parametrize("num_trials", [10, 50])
+@pytest.mark.parametrize("num_trials", [1, 10, 50])
 @pytest.mark.parametrize("num_dim", [2])
-@pytest.mark.parametrize("method", ["SNPE", "SNLE", "SNRE"])
+@pytest.mark.parametrize("method", ("SNPE",))
```

Comment:
What is the reason to remove SNLE and SNRE?

Reply:
they were misplaced here. the test is for the iid embeddings (for SNPE). iid inference with likelihood-based methods is already tested in `linear_gaussian_snle.py` etc.

Reply:
makes sense, thanks!